### DISCUSSION TITLE
Variogram generator in Python

### DISCUSSION THREAD

**Discussion by CooperGarth** [gmdsi/GMDSI_notebooks#154] [General]
@jtwhite79 I attended the PEST course in Perth a few weeks ago. 

You showed a Notebook where you could side different values to generate different fields of what Geostructures can look like and I think it's a good way to show what different Variogram look like.  

Are you able to upload that notebook?

**Comment by jtwhite79** [comment]
Eventually I think we want to integrate the`PyPestUtils` into the gmdsi notebooks, but for now that variogram-stochastic field demo is in an obscure location:

[https://github.com/jtwhite79/pypestutils/blob/develop/examples/understanding_variograms_and_realizations.ipynb](https://github.com/jtwhite79/pypestutils/blob/develop/examples/understanding_variograms_and_realizations.ipynb)

**Comment by CooperGarth** [comment]
Thanks @jtwhite79.
FYI for anyone else who uses the work book if you get a java script error refer to this page:
https://stackoverflow.com/questions/78544277/failed-to-display-jupyter-widgets
